e2e-tests/custom-login/specs/custom-login-flow-spec.js

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+const LoginHomePage = require('../../page-objects/shared/login-home-page');
+const CustomSignInPage = require('../../page-objects/custom-signin-page');
+const AuthenticatedHomePage = require('../../page-objects/shared/authenticated-home-page');
+const ProfilePage = require('../../page-objects/shared/profile-page');
+
+describe('Custom Login Flow', () => {
+  const loginHomePage = new LoginHomePage();
+  const customSignInPage = new CustomSignInPage();
+  const authenticatedHomePage = new AuthenticatedHomePage();
+  const profile = new ProfilePage();
+
+  beforeEach(() => {
+    browser.ignoreSynchronization = true;
+  });
+ 
+  it('can login with Okta as the IDP using custom signin page', async () => {
+    browser.get('http://localhost:8080/');
```

Comment:
Can we pass these static values (host, username, password) through configuration, with the option to pass it from the shell environment?  I know we're planning to use a common org on Travis, but being able to override is useful for local dev (and in case we ever need to change the details about the common test org).

Reply:
Adding support to pass them in protractor configuration.